### PR TITLE
Update documentation for scrollUntilVisible and friends.

### PR DESCRIPTION
**Pull request by Hixie**
Also minor code reformat.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: packages/flutter_test/lib/src/controller.dart [hunk 1]
```diff
@@ -1096,36 +1096,42 @@ abstract class WidgetController {
...
-  /// the screen correctly.
+  /// Usually the `finder` for this method should be labeled `skipOffstage:
+  /// false`, so that the [Finder] deals with widgets that are off the screen
+  /// correctly.
   ///
-  /// This does not work when the `S` is long and `W` far away from the
-  /// displayed part does not have a cached element yet. See
-  /// https://github.com/flutter/flutter/issues/61458
+  /// This does not work when `S` is long enough, and `W` far away enough from
+  /// the displayed part does of `S`, that `S` has not yet cached `W`'s element
```
Comment (goderbauer) [inline packages/flutter_test/lib/src/controller.dart:1104]: looks like there's still an extra "does" in this line?

File: packages/flutter_test/lib/src/controller.dart [hunk 2]
```diff
@@ -1096,36 +1096,42 @@ abstract class WidgetController {
...
-  /// [Scrollable.axisDirection] until `finder` is visible.
+  /// [Scrollable.axisDirection] direction until a widget matching `finder` is
+  /// visible.
   ///
-  /// Between each scroll, wait for `duration` time for settling.
+  /// Between each scroll, advances the clock by `duration` time.
   ///
-  /// If `scrollable` is `null`, this will find a [Scrollable].
+  /// Scrolling is performed until the start of the `finder` is visible. This is
+  /// due to the default parameter values of [Scrollable.ensureVisible] method.
```
Comment (goderbauer) [inline packages/flutter_test/lib/src/controller.dart:1120]: nit: ... of _the_ [Scrollable.ensureVisible] method.

**Review by goderbauer — APPROVED** [review]
LGTM

File: packages/flutter_test/lib/src/controller.dart (hunk 1 above)
Comment (Hixie) [reply packages/flutter_test/lib/src/controller.dart:1104]: wow that whole sentence was a disaster. fixed.

File: packages/flutter_test/lib/src/controller.dart (hunk 2 above)
Comment (Hixie) [reply packages/flutter_test/lib/src/controller.dart:1120]: fixed

**Pull request merged**